src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -217,6 +217,17 @@ default String getBasesString() {
     void setBaseQualities( final byte[] baseQualities );
 
     /**
+     * @return Base qualities string as printable ASCII values.
+     */
+    String getBaseQualityString();
+
+    /**
+     * Set the base qualities from a string of ASCII encoded values
+     * @param baseQualities - ASCII encode values of base qualities.
+     */
+    void setBaseQualityString( final String baseQualities );
```

Comment:
After some discussion with DR, we decided that these 2 methods probably aren't a great idea to include in the `GATKRead` interface because it might encourage improper use of the the base qualities as strings.  Could you move modify them so they accept a `GATKRead` as a parameter and move them to Read utils?